Plugins/GitHub3/GitHub3Plugin.cs

```diff
@@ -101,7 +104,37 @@ public GitHub3Plugin() : base(true)
 
         public override IEnumerable<ISetting> GetSettings()
         {
-            yield return OAuthToken;
+            yield return PersonalAccessToken;
+
+            var generateTokenLink = new LinkLabel { Text = _generateToken.Text };
+            generateTokenLink.Click += GenerateTokenLink_Click;
+            yield return new PseudoSetting(generateTokenLink);
+
+            var manageTokenLink = new LinkLabel { Text = _manageToken.Text };
+            manageTokenLink.Click += ManageTokenLink_Click;
+            yield return new PseudoSetting(manageTokenLink);
+        }
+
+        private void GenerateTokenLink_Click(object sender, EventArgs e)
+        {
+            OpenLink($"https://{GitHubHost.ValueOrDefault(Instance.Settings)}/settings/tokens/new?description=Token%20for%20GitExtensions&scopes=repo,public_repo");
+        }
+
+        private void ManageTokenLink_Click(object sender, EventArgs e)
+        {
+            OpenLink($"https://{GitHubHost.ValueOrDefault(Instance.Settings)}/settings/tokens");
+        }
+
+        private void OpenLink(string url)
+        {
+            try
+            {
+                Process.Start(url);
```

Comment:
At least for .NET 5, `useShellExecute: true` and `Verb = useShellExecute` are needed.

Reply:
@mstv I did something. Is that good? Because I don't understand the `Verb = useShellExecute` part and didn't found something about it. Do you have a documentation?

Reply:
👍, just add the `Verb` like here:
https://github.com/gitextensions/gitextensions/blob/d488e497cc6244483794abedb8e4779495ba861c/GitCommands/Git/Executable.cs#L83-L101

Reply:
Sorry, I have corrected a copy & paste mistake in my initial comment.

Reply:
@mstv Done. Even if it works without...